src/Microsoft.Data.Analysis/DataFrameColumn.cs

```diff
@@ -111,10 +112,11 @@ public virtual DataFrameColumn Sort(bool ascending = true)
         public virtual GroupBy GroupBy(int columnIndex, DataFrame parent) => throw new NotImplementedException();
 
         /// <summary>
-        /// Returns a new column with nulls replaced by value
+        /// Returns a new column with nulls replaced by <paramref name="value"/>
         /// </summary>
         /// <remarks>Tries to convert value to the column's DataType</remarks>
         /// <param name="value"></param>
+        /// <param name="inPlace">Indicates if the operation should be performed in place</param>
```

Comment:
Any exceptions?

Reply:
Oh, there are. There's undocumented exceptions in a bunch of other methods too. I wonder if there's a static analysis tool that can look at all possible code paths and generate all possible exceptions in the documentation.